ebscohost/list.json

```diff
@@ -422,7 +424,6 @@
   "E866SWW": "Middle Search Reference eBook Subscription",
   "MTH": "Military & Government Collection",
   "MLE": "Military and Law Enforcement Periodicals, 1691-1877",
-  "MYT": "Military Transition Support Center",
```

Comment:
It is not recommended to delete this line, because if we do a new ezPAARSE job in which this line appears in the log, it will no longer be enriched